vendor/jasmine-only/jasmine-only.js

```diff
@@ -0,0 +1,98 @@
+/* jasmine-only - 0.1.1
+ * Exclusivity spec helpers for jasmine: `describe.only` and `it.only`
+ * https://github.com/davemo/jasmine-only
+ */
+(function() {
```

Comment:
I'm unsure whether you want this library only to run in a node 4 environment. It seems to me like that wouldn't be worth it – what do you think?